Examples/Algorithms/Digitization/src/DigitizationAlgorithm.cpp

```diff
@@ -206,6 +207,19 @@ ActsExamples::ProcessCode ActsExamples::DigitizationAlgorithm::execute(
             const auto& simHit = *h;
             const auto simHitIdx = simHits.index_of(h);
 
+            const auto hitIndex =
+                particleOnModuleHitCount[simHit.particleId()]++;
+
+            if (hitIndex > 0) {
+              ACTS_DEBUG("more than one hit for particle "
+                         << simHit.particleId() << " on module " << moduleGeoId
+                         << " - still forwarding it to the digitizer");
+            }
+
+            auto hitSeed = eventSeed + moduleGeoId.value() +
```

Comment:
Are we concerned about overflow with `value` possibly having very large values? Also, do we need the `auto` or can we be more specific here?

Reply:
Our seeds are `std::uint64_t` so they should gracefully wrap around. I didn't want to spell out the specific type but maybe I can typedef it